tests/lib/rules/no-jquery.js

```diff
@@ -26,7 +26,7 @@ eslintTester.run('no-jquery', rule, {
         });
 
         test('assert something', function() {
-          assert.equal(find('.some-component').textContent.trim(), 'hello world');
+          assert.strictEqual(find('.some-component').textContent.trim(), 'hello world');
```

Comment:
I don't understand this change. This is QUnit API and unrelated to Node.js APIs 🤔

Reply:
Haha oops, fixed.